src/mpid/ch4/generic/gpu/mpidig_gpu_utils.h

```diff
@@ -5,8 +5,6 @@
 #ifndef MPIDIG_GPU_UTIL_H_INCLUDED
 #define MPIDIG_GPU_UTIL_H_INCLUDED
 
-#include "ch4_impl.h"
```

Comment:
Why is this removed?  We need `mpiimpl.h` or something here.  Also, this seems unrelated to this PR.

Reply:
The functions in this file are from `ch4_impl.h` anymore, this is a leftover remove from https://github.com/pmodels/mpich/pull/4528. Leaving include `ch4_impl.h` here would cause recursive header inclusion issues. Because request completion in `ch4_impl.h` calls functions here.

Reply:
Can you separate out that patch with the appropriate commit message explaining it?

Reply:
Will do.